bin/rngen

```diff
@@ -215,7 +215,7 @@ for line in "${changelog_json[@]}"; do
   user="$(echo "$line" | jq -r .user.login)"
   body="$(echo "$line" | jq -r .body)"
   gh_url="$(echo "$line" | jq -r .html_url)"
-  jira_issue="$(echo "$body" | grep -oE '[A-Z]{2,8}-[0-9]{1,6}' | head -1)"
+  jira_issue="$(echo "$body $title" | grep -oE '[A-Z]{2,8}-[0-9]{1,6}' | head -1)"
```

Comment:
Want to put the title first? Perhaps if there's a ticket ref in the title, it should be prioritized.

Reply:
sure 👍